### PR TITLE
Display output using template

### PR DESCRIPTION
**Pull request by utkarshmani1997**
* Local variable naming as prescribed by the [Golang community](https://github.com/golang/go/wiki/CodeReviewComments#variable-names).
* Displays output using Template.

### CODE REVIEW

File: command/vsm_stats.go [hunk 1]
```diff
@@ -427,17 +428,17 @@ func StatsOutput(c *VsmStatsCommand, annotations *Annotations, args []string, st
 
 	} else {
 
-		// Printing in tabular form
-		//	fmt.Printf("%+v\n\n", annotation)
-		data, err := json.MarshalIndent(annotation, "", "\t")
-
+		// Printing using template
+		tmpl, err := template.New("test").Parse("IQN     : {{.IQN}}\nVolume  : {{.Volume}}\nPortal  : {{.Portal}}\nSize    : {{.Size}}")
```
Comment (AmitKumarDas) [inline command/vsm_stats.go]: Use typed string constant for the string used in template.
Eventually this typed string will be used to define many more templates.

File: command/vsm_stats.go [hunk 2]
```diff
@@ -427,17 +428,17 @@ func StatsOutput(c *VsmStatsCommand, annotations *Annotations, args []string, st
...
-
 			panic(err)
 		}
 
-		os.Stdout.Write(data)
-
+		// Printing in tabular form
 		q := tabwriter.NewWriter(os.Stdout, minwidth, maxwidth, padding, ' ', tabwriter.AlignRight|tabwriter.Debug)
 
 		fmt.Fprintf(q, "\n\nReplica\tStatus\tDataUpdateIndex\t\n")
```
Comment (AmitKumarDas) [inline command/vsm_stats.go]: Why are we not using template feature vs. fmt.Fprintf ?

Comment (utkarshmani1997) [inline command/vsm_stats.go]: This is required to print the output in tabular format as suggested by @prateekpandey14.

Comment (utkarshmani1997) [inline command/vsm_stats.go]: this is the syntax of tabwriter.

**Comment by utkarshmani1997** [comment]
the changes are merged in #83 .Closing this PR

**Pull request closed without merging**